### ISSUE TITLE
CMD+K not working

### ISSUE THREAD

**Issue by oliviervdv** [SethuSenthil/Copy-for-ChatGPT#15]
The keyboard shortcut is currently not working. This may be due to a change in ChatGPT. I just wanted to notify you in case you were not aware. Great extension, it has saved me so much time...

**Comment by SethuSenthil** [comment]
Thank you for reporting this issue, I was not aware of this! Will work on a fix!

**Comment by dracount** [comment]
Yes - CTRL + K does something different in windows - it accesses the address bar

**Comment by oliviervdv** [comment]
is there any evolution on this? Would love to see it back up and running :D
